spec/time_of_day_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Timely::TimeOfDay do
+  subject(:time_of_day) { Timely::TimeOfDay.new(4, 52) }
+
+  its(:hour) { should == 4 }
+  its(:minute) { should == 52 }
+
+  context 'asking for that time on a date' do
+    subject(:time) { time_of_day.on_date(Date.new(2014, 1, 1)) }
+    its(:month) { should == 1 }
+    its(:hour) { should == 4 }
+    its(:min) { should == 52 }
+  end
+
+  context 'can initialize from a time' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.